psi4/driver/driver_findif.py

```diff
@@ -238,13 +245,72 @@ def _geom_generator(mol, freq_irrep_only, mode):
 
     Returns
     -------
-    disp_geoms : list of psi4.core.Matrix
-        A list of displaced geometries to compute quantities at.
+    findifrec : dict
+        Dictionary of finite difference data, specified below.
+        The dictionary makes findifrec _extensible_. If you need a new field
+        in the record, just add it.
+        All fields should be present at all times, with two exceptions:
+            1. Fields for computed quantities will not be available until
+               after they are computed.
+            2. Displacement specific overrides for globals will not be
+               available unless the user specified the overrides.
+               (Such overrides are not implemented at time of writing. An example
+               is giving a displacement its own step dict.)
+
+    step : dict
+        A descriptor for the finite difference step.
+        In future, this can be overriden by step fields for individual displacements.
+
+        units : {'bohr', 'angstrom'}
```

Comment:
for cohesion with everything else that started from libmints, make this `{'Bohr', 'Angstrom'}`